css/types/frequency-percentage.json

```diff
@@ -0,0 +1,160 @@
+{
+  "css": {
+    "types": {
+      "frequency-percentage": {
+        "__compat": {
+          "description": "<code>&lt;frequency-percentage&gt;</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/frequency-percentage",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "hz": {
+        "__compat": {
+          "description": "<code>Hz</code> unit",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
I'm still trying to learn how to find CSS support in Chrome myself. This suggests this should be true: https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/css/css_primitive_value_units.json5

The file is newer than the feature, so I can't get you a version number.

Reply:
I can't find a version, I just took this from the support listed for frequency on MDN, but it does appear to be in the code so have updated both files.